mesh/mesh.hpp

```diff
@@ -990,6 +990,12 @@ class Mesh
        for example, after the mesh nodes are modified externally. */
    void DeleteGeometricFactors();
 
+   /// @brief This function should be called after the mesh node coordinates
+   /// have changed, e.g. after the mesh has moved.
+   /** It updates internal quantities derived from the node coordinates, such as
```

Comment:
```suggestion
   /** It updates internal quantities derived from the node coordinates, such
```

Sorry, I introduced a typo 🙈